src/main/java/com/vasivkov/bootstrap/H2JdbcConnectionPool.java

```diff
@@ -26,11 +27,12 @@ private H2JdbcConnectionPool() {
     public void init() {
         if (jdbcConnectionPool == null) {
             jdbcConnectionPool = JdbcConnectionPool.create(H2_URL, H2_NAME, H2_PASSWORD);
+            LOGGER.info("jdbcConnectionPool was successfully created");
             jdbcConnectionPool.setMaxConnections(10);
             try {
                 jdbcConnectionPool.getConnection();
             } catch (SQLException e) {
-                e.printStackTrace();
+                LOGGER.error("Failed to create connection to H2");
```

Comment:
It's good to provide exception as a second argument for 'error' method. Using this you will not lose stacktrace.